samtranslator/model/sam_resources.py

```diff
@@ -624,7 +627,9 @@ def _construct_role(
         )
 
         managed_policy_arns = [ArnGenerator.generate_aws_managed_policy_arn("service-role/AWSLambdaBasicExecutionRole")]
-        if self.Tracing:
+
+        tracing = intrinsics_resolver.resolve_parameter_refs(self.Tracing)
+        if tracing == TRACING_CONFIG_ACTIVE:
```

Comment:
This will likely cause redeployment right? Since anyone so far that has used `Tracing` to something other than `Active` will now have the policy removed.

Reply:
hmmm actually good point. It would.